tools/stonith_admin.c

```diff
@@ -616,14 +577,30 @@ main(int argc, char **argv)
     }
 
     if (required_agent && agent == NULL) {
-        printf("Please specify an agent to query using -a,--agent [value]\n");
+        fprintf(stderr, "Please specify an agent to query using -a,--agent [value]\n");
         ++argerr;
     }
 
     if (argerr) {
         crm_help('?', CRM_EX_USAGE);
     }
 
+    CRM_ASSERT(pcmk__register_format("text", pcmk__mk_text_output) == 0);
+    CRM_ASSERT(pcmk__register_format("xml", pcmk__mk_xml_output) == 0);
+
+    rc = pcmk__output_new(&out, output_ty, output_dest, argv);
+    if (rc == ENXIO) {
+        fprintf(stderr, "Unknown format given for --output-as: %s\n", output_ty);
+        exit_code = CRM_EX_ERROR;
+        goto done;
+    } else if (rc != 0) {
+        fprintf(stderr, "Could not create/initialize output format: %s\n", pcmk_strerror(rc));
+        exit_code = CRM_EX_ERROR;
+        goto done;
+    }
```

Comment:
Not necessary for this PR, but this if-else indicates that we'd benefit from a pcmk_err_unknown_format code, so pcmk_strerror() is sufficient

Reply:
Might as well do it now.  It's minimal work.